### PR TITLE
docs: initialize README

### LINKED ISSUE #24: initialize README

### PR DESCRIPTION
**Pull request by zi-gae**
리드미 생성

- closed #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,106 @@
...
+이러쿵 저러쿵 사용하시면 됩니다
+
+## 🚀 Deployment <a name = "deployment"></a>
+
+배포.. 메이비.. 도커로.. 할거니까.. ec2 아니면 fargate.. CD 구축해서.. 할.. 것.. 같.. 은.. 데..?
+
+## ⛏️ Built Using <a name = "built_using"></a>
+
+- [Next](https://nextjs.org/) - Web Framework
+- [Express](https://expressjs.com/) - Server Framework
```
Comment (Yuni-Q) [inline README.md:96]: express ??

Comment (zi-gae) [reply README.md:96]: Express inside Next 🤔

Comment (Yuni-Q) [reply README.md:96]: next don't have express 😅

Comment (Yuni-Q) [reply README.md:96]: Do we plan to use express?

Comment (zi-gae) [reply README.md:96]: I thought had Express on `api` folder. my mistake. 암 뉴비 of Nextjs  😂

Comment (zi-gae) [reply README.md]: https://github.com/Nexters/TILTIL_FRONT/pull/38/commits/f702443ad2e56b1a4849ed6f33efc18f7d774e1d Thank you 👍

Comment (zi-gae) [reply README.md]: Switch to resolved

**Review by Yuni-Q — APPROVED** [review]

**Pull request merged**